readme/BREAKING_CHANGES.markdown

```diff
@@ -480,4 +480,26 @@ The `frontend-taglib-clay` module is now using components from
 [`Clay v3`](https://github.com/liferay/clay), which doesn't support the
 previous attributes.
 
+---------------------------------------
+### Remove CSS Compatibility Layer
+- **Date:** January 2021
+- **JIRA Ticket:** [LPS-123359](https://issues.liferay.com/browse/LPS-123359)
+
+#### What changed?
+
+The support for `BS3 markup` has been deleted and is no longer available.
```

Comment:
Should we spell out "Bootstrap" here, in case people don't know what BS means?

Also, I am not sure why we need backticks here. Those would usually be for code, and you are not using them in the other places where you say "BS3 markup" or "BS4 markup".

Reply:
done!